en/docs/SetupAndInstall/DeployingWSO2APIManager/SingleNode/configuring-an-active-active-deployment.md

```diff
@@ -23,96 +21,87 @@ Create a SSL certificate on the first WSO2 API-M all-in-one active node. For mor
 
 ### Step 2 - Configure the load balancer
 
-For information on configuring the load balancer, see \_Configuring the Proxy Server and the Load Balancer .
+For information on configuring the load balancer, see [Configuring the Proxy Server and the Load Balancer](../configuring-the-proxy-server-and-the-load-balancer.md).
 
 ### Step 3 - Configure the databases
 
-For information on configuring the databases, see \_Installing and Configuring the Databases .
+For information on configuring the databases, see [Installing and Configuring the Databases](../installing-and-configuring-the-databases.md).
 
 ### Step 4 - Configure the Publisher with the Gateway
 
 !!! note
-This step is required **only** if you are using rsync to share files.
-
-!!! info
-When you use rsync the file synchronization will happen in only one direction. Therefore, use the following configuration to enable synchronization in both directions between two nodes.
-
+    This step is required **only** if you are using rsync to share files.
 
-For more details, see the [next step](#ConfiguringanActive-ActiveDeployment-Step5) .
+    !!! info
+        When you use rsync the file synchronization will happen in only one direction. Therefore, use the following configuration to enable synchronization in both directions between two nodes.
 
+    For more details, see the [next step](#ConfiguringanActive-ActiveDeployment-Step5-Configurethecontentsynchronizationmechanism).
 
 Configure the API Publisher in both nodes to be able to publish to the API-M Gateway of one of the nodes. Do this by pointing the `<ServerURL>` to the same Gateway node.
 
 You need to configure this in the `<API-M_HOME>/repository/conf/api-manager.xml` file.
 
 ``` java
-    <APIGateway>
-        <ServerURL>https://localhost:${mgt.transport.https.port}${carbon.context}services/</ServerURL>
-    </APIGateway>
+<APIGateway>
```

Comment:
Shouldn't we have the new config model here?

Reply:
Yes, I am rewriting that part. Will put a separate PR addressing the changes.